src/containers/TriggerTemplate/TriggerTemplate.js

```diff
@@ -0,0 +1,377 @@
+
+        <div className="controlsDiv">
+          <Button onClick={() => this.handleDetails()} kind="ghost">
+            {intl.formatMessage({
+              id: 'dashboard.triggerTemplate.details',
+              defaultMessage: 'Details'
+            })}
+          </Button>
+          <Button onClick={() => this.handleYAML()} kind="ghost">
+            {intl.formatMessage({
+              id: 'dashboard.triggerTemplate.yaml',
+              defaultMessage: 'YAML'
+            })}
+          </Button>
+        </div>
+
+        {this.state.showYAML && (
+          /* eslint-disable react/no-unknown-property */
+          <div className="textarea">
+            <pre id="yaml">
+              <p className="triggertemplatetext">
+                <p>{jsYaml.dump(triggerTemplate)}</p>
+              </p>
+            </pre>
+          </div>
+        )}
+
+        {this.state.showDetails && (
+          /* eslint-disable react/no-unknown-property */
+          <div className="textarea">
+            <pre id="yaml">
+              <p className="triggertemplatetext">
+                <p>
+                  {intl.formatMessage({
+                    id: 'dashboard.triggerTemplate.creationTime',
+                    defaultMessage: 'Creation time'
+                  })}
+                  : {triggerTemplateDate}
+                </p>
+                <p>
+                  {intl.formatMessage({
+                    id: 'dashboard.triggerTemplate.namespace',
+                    defaultMessage: 'Namespace'
+                  })}
+                  : {triggerTemplateNamespace}
+                </p>
+
+                {triggerTemplateLabels && (
+                  <p>
+                    <>
+                      {intl.formatMessage({
+                        id: 'dashboard.triggerTemplate.labels',
+                        defaultMessage: 'Labels'
+                      })}
+                      : ${triggerTemplateLabels}
+                    </>
+                  </p>
+                )}
+              </p>
+            </pre>
+          </div>
+        )}
+
+        <DashboardTable
+          title="Parameters"
+          headers={headersForParameters}
+          rows={rowsForParameters}
+          selectedNamespace={selectedNamespace}
+          emptyTextAllNamespaces={intl.formatMessage(
+            {
+              id: 'dashboard.emptyState.allNamespaces',
+              defaultMessage: 'No {kind} under any namespace.'
+            },
+            { kind: 'TriggerTemplates' }
+          )}
+          emptyTextSelectedNamespace={intl.formatMessage(
+            {
+              id: 'dashboard.emptyState.selectedNamespace',
+              defaultMessage: 'No {kind} under namespace {selectedNamespace}'
+            },
+            { kind: 'TriggerTemplate', selectedNamespace }
+          )}
+          isSortable
+        />
+
+        {rowsForParameters.length === 0 && (
+          <h3>No parameters found for this TriggerTemplate</h3>
```

Comment:
is this the same as emptyTextAllNamespaces?

Reply:
Yep removed